projectional/src/main/java/jetbrains/jetpad/projectional/cell/BaseProjectionalSynchronizer.java

```diff
@@ -66,6 +70,8 @@
   private DeleteHandler myDeleteHandler = DeleteHandler.EMPTY;
   private ContentKind<SourceItemT> myItemKind;
   private Function<SourceItemT, SourceItemT> myCloner;
+  private ListMap<ContentKind, ContentToItemMapper> myCompatibleContentKinds = new ListMap<>();
+  private ListMap<ContentKind, ContentToItemsListMapper> myContentKindsCompatibleToItemsList = new ListMap<>();
```

Comment:
"myContentKindsCompatibleToItemsList" is long and not clear. I would suggest "myListContentKinds".